### DISCUSSION TITLE
Can't login to Fusion360

### DISCUSSION THREAD

**Discussion by Connor9220** [cryinkfly/Autodesk-Fusion-360-for-Linux#325] [General]
Fusion 360 is trying to start up a browser to get past the login screen.  But, nothing it happening?  What am I missing?  I removed my previous install and re-installed because I needed an updated version.

**Comment by apos** [comment]
I have the exact same problem. Installed with flatpak bottle. So do I need to install a browser in the bottle?

**Comment by apos** [comment]
See https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/issues/319#issuecomment-1504925307

**Comment by escaflowne52** [comment]
The problem persists, I can't connect Fusion360 either. The web browser opens fine, but when I log on to the site, a message tells me I'm going to switch back to the application, but nothing happens...
I'm on Debian 12 with a KDE environment.

**Comment by jeromio** [comment]
I am stuck here as well - although it *was* working on my system. I guess updating Wine (to 10.15) broke it. But I tried downgrading to 10.14 and 10.13 and it does the same thing there as well. My system shows the adskidentitymanager.exe running, but the browser wants to direct to adskidmgr.exe, which is not present.  Not really knowing how this all is supposed to work, I tried pointing it directly to that adskidentitymanager.exe within .autodesk_fusion folder, but that doesn't work either. Just hangs there.
